activeadmin.gemspec

```diff
@@ -22,10 +22,10 @@ Gem::Specification.new do |s|
   s.add_dependency 'arbre', '~> 1.1', '>= 1.1.1'
   s.add_dependency 'formtastic', '~> 3.1'
   s.add_dependency 'formtastic_i18n', '~> 0.4'
-  s.add_dependency 'inherited_resources', '~> 1.7'
+  s.add_dependency 'inherited_resources'
```

Comment:
Is this change needed? Master branch of inherited resources is still compatible with this requirement I think.

Reply:
yes, this is actually not needed
I added it back